lib/chef/knife/kvm_base.rb

```diff
@@ -33,6 +33,7 @@ def run_remote_command(command)
               if success
                 channel.on_data do |ch, data|
                   return_val = data.chomp
+                  print return_val if console
```

Comment:
Ah OK. New lines might mess things up yes :smile: 

Looks like we can grab the `stdout` directly from the `Ui` class. https://github.com/chef/chef/blob/master/lib/chef/knife/core/ui.rb#L35 We can maybe directly `write` to it. 

Either that or we can